src/test/groovy/com/meltwater/docker/compose/DockerComposeTest.groovy

```diff
@@ -53,7 +53,7 @@ class DockerComposeTest extends Specification {
         given:
             compose.up()
         when:
-            compose.kill("composetest_foo_1")
+            compose.kill("compose_test_foo_1")
```

Comment:
Same here, changing on line 12 is better.

Reply:
sounds good, let me push the change you suggest ...